tools/public_api_guard/material/timepicker.md

```diff
@@ -25,6 +26,9 @@ import { Validator } from '@angular/forms';
 // @public
 export const MAT_TIMEPICKER_CONFIG: InjectionToken<MatTimepickerConfig>;
 
+// @public
+export const MAT_TIMEPICKER_SCROLL_STRATEGY: InjectionToken<() => ScrollStrategy>;
```

Comment:
is there a reason to not just put it as part of the `MAT_TIMEPICKER_CONFIG`?

Reply:
Mostly consistency with the rest of the components.